src/login/Login.jsx

```diff
@@ -25,21 +25,26 @@ export default function Login() {
             },
             body: JSON.stringify(payload)
         };
-
         fetch("http://127.0.0.1:5000/login", requestLogin)
             .then(async response => {
                 let data = await response.json();
                 if (response.status === 200) {
+                    if (data !== {}) {
```

Comment:
Is your purpose to see if data is an empty object?
Please look into considerations for such tests, and I would recommend using one of the suggestions from here: https://stackoverflow.com/questions/679915/how-do-i-test-for-an-empty-javascript-object

Reply:
Another resource: https://stackoverflow.com/questions/4994201/is-object-empty

Reply:
Thanks, @ramitsawhney27 . I've changed it to one of the suggested option (using underscore library _.isEmpty(obj)) 👍